package-lock.json

```diff
@@ -1428,6 +1401,12 @@
       "integrity": "sha512-kssjab8CvdXfcXMXVcvsXum4Hwdq9XGtRD3TteMEvEbq0LXyiNQr6AprqKqfeaDXze7SxWvRxdpwE6ku7ikLkg==",
       "dev": true
     },
+    "html-escaper": {
```

Comment:
what is this dependency for?

Reply:
It comes from the update of the "istanbul-reports" (I don't know what that stands for).   The requirements have been changed from "handlebars": "^4.1.2" to "html-escaper" changed: "^2.0.0".